src/main/flight/gps_rescue.c

```diff
@@ -536,6 +538,12 @@ void updateGPSRescueState(void)
             newDescentDistanceM = gpsRescueConfig()->descentDistanceM;
         }
         
+        if (gpsRescueConfig()->allowClimbToMaxReachedAlt) {
```

Comment:
Why was this moved to here?

Reply:
For non copy/paste it in the two cases (`RESCUE_ATTAIN_ALT` and `RESCUE_CROSSTRACK`